packages/lib-utils/src/app/redux/index.ts

```diff
@@ -25,7 +25,15 @@ type UseReduxStoreResult = {
  * ```
  */
 export const useReduxStore = (): UseReduxStoreResult => {
-  const storeContext = useStore();
+  let storeContext: Store | null = null;
+  try {
+    // It'll always be invoked -- it just might blow up
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    storeContext = useStore();
+  } catch (e) {
+    // TODO: Handle this better...
+    consoleLogger.error(e);
+  }
```

Comment:
I don't think we need this anymore (given we update the redux to same version hosted from hac-core). But let's keep it here and remove later down the line. Can you add `TODO: remove once proven not needed`?